charts/sn-platform/templates/vault/vault-clear-resource.yaml

```diff
@@ -37,6 +37,13 @@ spec:
     metadata:
       labels:
         {{- include "pulsar.template.labels" . | nindent 8 }}
+{{- if .Values.vault.labels }}    
+{{ toYaml .Values.vault.labels | indent 8 }}
+{{- end }}
+      annotations:
+{{- if .Values.vault.annotations }}    
```

Comment:
Will it be a problem if `.Values.vault.annotations` is nil?

Reply:
You mean if the values config set like? 

```
vault:
  annotations: {}
```